src/utilities/getNumericalTarget.ts

```diff
@@ -0,0 +1,30 @@
+import max from 'ml-array-max';
+import min from 'ml-array-min';
+
+import { ClassType } from '../../types/Class';
+
+/**
+ * Returns the array of metadata numerically categorized.
+ * @param {Array} target Array containing the categories.
+ * @param {Array} test Array containing the result of prediction.
+ * @param {Object} pair Object with information about two classes.
+ * @return {Array<number>} Array containing the categories assinged as numbers.
+ */
+
+export function getNumericalTarget(
+  target: string[],
+  test: number[],
+  pair: ClassType[],
+) {
+  const boundaries = [min(test) - 0.001, max(test) + 0.001];
+  let result: number[] = [];
```

Comment:
We should try to have 'plural' if a variable contains an array. Please use `results` as variable name